build-tools/packages/build-tools/src/bumpVersion/utils.ts

```diff
@@ -46,3 +48,111 @@ export function prereleaseSatisfies(packageVersion: string, range: string) {
     // Pretend that the current package is latest prerelease (zzz) and see if the version still satisfies.
     return semver.satisfies(`${packageVersion}-zzz`, range)
 }
+
+/**
+ * Translate a VersionChangeType for the virtual patch scenario where we overload a beta version number
```

Comment:
Nit
```suggestion
 * Translate a {@link VersionChangeType} for the virtual patch scenario where we overload a beta version number
```

At least in theory should help prevent doc staleness if we use VSCode refactoring tools